forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -4571,7 +4582,28 @@ public Table<Long, String, KeywordInterface> getStoredKeywords() {
     public void setStoredKeywords(Table<Long, String, KeywordInterface> table, boolean lki) {
         storedKeywords.clear();
         for (Table.Cell<Long, String, KeywordInterface> c : table.cellSet()) {
-            storedKeywords.put(c.getRowKey(), c.getColumnKey(), c.getValue().copy(this, lki));
+            KeywordInterface stored = null;
+
+            // for performance check if we already copied this
+            if (lki) {
+                for (Table.Cell<Long, Long, KeywordsChange> kwCell : changedCardKeywords.cellSet()) {
+                    // same static
+                    if (kwCell.getColumnKey().equals(c.getRowKey())) {
+                        for (KeywordInterface kw : kwCell.getValue().getKeywords()) {
+                            // same kw
+                            if (kw.getOriginal().equals(c.getValue().getOriginal())) {
+                                stored = kw;
+                            }
+                        }
```

Comment:
i don't think that is always true what if the card would give multiple instances of the same keyword?

Reply:
L4591 does check that the static IDs are same too

I mean otherwise ``getKeywordForStaticAbility()`` would have a problem too?